Godeps/Godeps.json

```diff
@@ -147,18 +147,38 @@
 			"Rev": "bbeaab5ee38568daa84b8c233a8067b524588355"
 		},
 		{
+			"ImportPath": "github.com/containernetworking/cni/libcni",
+			"Comment": "v0.3.0-61-g9d5e6e6",
```

Comment:
This is imported by the `cri-o`, so we I think we should let the version be even with the version which `cri-o` is using.